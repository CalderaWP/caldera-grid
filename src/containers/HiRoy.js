import React from 'react';
import {Thing} from "../components/Thing";

export const HiRoy = (props) => {
    return (
        <div>
            <Thing.Edit
                label={props.label}
                onChange={props.onChange}
                idAttr={props.idAttr}
                thing={props.thing}
            />
            <Thing.Display
                thing={props.thing}
            />
        </div>
    )
};

HiRoy.propTypes = Thing.propTypes;

