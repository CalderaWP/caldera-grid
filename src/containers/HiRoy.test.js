import React from 'react';
import {HiRoy } from "./HiRoy";
import renderer from 'react-test-renderer';

test('Test with message ', () => {
    const component = renderer.create(
        <HiRoy
            thing={'Hi Roy'}
            idAttr={'thing-chooser'}
            label={'Change Thing'}
            onChange={(event) => {}}
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

