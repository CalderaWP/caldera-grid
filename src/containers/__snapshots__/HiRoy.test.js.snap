// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test with message  1`] = `
<div>
  <div>
    <label
      htmlFor="thing-chooser"
    >
      Change Thing
    </label>
    <input
      id="thing-chooser"
      onChange={[Function]}
      value="Hi Roy"
    />
  </div>
  <div>
    Hi Roy
  </div>
</div>
`;
